### PR TITLE
Add Prow setup for feature flag tests

### PR DESCRIPTION
**Pull request by JeromeJu**
# Changes
This commit adds the prow setup for feature flag tests.

/kind misc
fixes: #1629

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -2085,3 +2085,57 @@ periodics:
       env:
       - name: TEST_RUN_ALL_TESTS
         value: "true"
+- cron: "6 4 * * *"
```
Comment (dibyom) [inline prow/config.yaml:2088]: @JeromeJu  curious why the cron time is so specific

Comment (JeromeJu) [reply prow/config.yaml:2088]: I intended to follow what's done by [periodic-tekton-catalog-integration-tests](https://github.com/tektoncd/plumbing/commit/ce9c2967903ea2dcd070a5c3cf826b04f44658eb) as it sets off the difference from [02:05 am](https://github.com/tektoncd/plumbing/blob/0cf9834f1672a1a14d4f377de86124df369040a8/prow/config.yaml#L1984) with 1 hr and 1 minute; which I interpret as it wants to have 60 minutes in between each cron job?

Comment (dibyom) [reply prow/config.yaml:2088]: ok, interesting

**Review by dibyom — APPROVED** [review]
cc @vdemeester @afrittoli - adding a new prow job. they might have an impact on CI/infra costs

**Review by vdemeester — APPROVED** [review]

**Comment by JeromeJu** [comment]
> /test plumbing-unit-tests

This unit test seems pretty broken 🤔 https://github.com/tektoncd/plumbing/issues/1804

**Pull request merged**